rts/lra/narayana-lra/src/main/java/io/narayana/lra/filter/ServerLRAFilter.java

```diff
@@ -331,6 +331,7 @@ public void filter(ContainerRequestContext containerRequestContext) {
                             toURI(terminateURIs.get(STATUS)),
                             null);
                 } catch (NotFoundException e) {
+                    lraTrace(containerRequestContext, lraId, "ServerLRAFilter before: not found with " + e.getMessage());
```

Comment:
```suggestion
                    lraTrace(containerRequestContext, lraId, "ServerLRAFilter before: not found exception with " + e.getMessage());
```

seems a little out of context without this word